common/lib/common-definitions/src/logger.ts

```diff
@@ -31,6 +40,13 @@ export interface ITelemetryBaseEvent extends ITelemetryProperties {
  * Implemented by hosting app / loader
  */
 export interface ITelemetryBaseLogger {
+    /**
+     * An optional boolean which indicates to the logger that tags (i.e. `ITaggedTelemetryPropertyType` objects) are
```

Comment:
"to the logger" -> "to user of this interface"

Reply:
Changed.